hubconf.py

```diff
@@ -41,7 +41,8 @@ def create(name, pretrained, channels, classes):
             model.load_state_dict(state_dict, strict=False)  # load
             if len(ckpt['model'].names) == classes:
                 model.names = ckpt['model'].names  # set class names attribute
-            # model = model.autoshape()  # for PIL/cv2/np inputs and NMS
+            if autoshape:
```

Comment:
I assumed that if a user pulls a yolov5 `pretrained=False` model, they will train it, therefore the autoshape parameter has no effect.